bot/cogs/slowmode.py

```diff
@@ -0,0 +1,86 @@
+import logging
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+from discord import TextChannel
+from discord.ext.commands import Cog, Context, group
+
+from bot.bot import Bot
+from bot.constants import Emojis, MODERATION_ROLES
+from bot.converters import DurationDelta
+from bot.decorators import with_role_check
+from bot.utils import time
+
+log = logging.getLogger(__name__)
+
+
+class Slowmode(Cog):
+    """Commands for getting and setting slowmode delays of text channels."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @group(name='slowmode', aliases=['sm'], invoke_without_command=True)
+    async def slowmode_group(self, ctx: Context) -> None:
+        """Get and set the slowmode delay for a given text channel."""
+        await ctx.send_help(ctx.command)
+
+    @slowmode_group.command(name='get', aliases=['g'])
+    async def get_slowmode(self, ctx: Context, channel: TextChannel) -> None:
+        """Get the slowmode delay for a given text channel."""
+        delay = relativedelta(seconds=channel.slowmode_delay)
+
+        # Say "0 seconds" instead of "less than a second"
+        if channel.slowmode_delay == 0:
+            humanized_delay = '0 seconds'
+        else:
+            humanized_delay = time.humanize_delta(delay)
```

Comment:
We shouldn't monkeypatch the functionality in our own utils inside of unrelated cogs. This is a code smell.

The change is fine, but if you're going to make this change it should be made inside of `utils.time`, not here.

Reply:
Yeah, that would make sense. Thanks for pointing this out.